src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -439,6 +450,18 @@ public boolean requiresIntervals() {
     }
 
     /**
+     * Load the master sequence dictionary as specified in {@code masterSequenceDictionaryFilename}.
+     * Will only load the master sequence dictionary if it has not already been loaded.
+     */
+    private void loadMasterSequenceDictionary() {
+        if ( masterSequenceDictionary == null ) {
+            if (masterSequenceDictionaryFilename != null) {
```

Comment:
Combine the two ifs: `if ( masterSequenceDictionary == null && masterSequenceDictionaryFilename != null )`